src/utils/scales-utils.js

```diff
@@ -268,12 +268,14 @@ function _collectScaleObjectFromProps(props, attr) {
     [`${attr}Range`]: range,
     [`${attr}Distance`]: distance = 0,
     [`${attr}BaseValue`]: baseValue,
-    [`${attr}Type`]: type = LINEAR_SCALE_TYPE} = props;
+    [`${attr}Type`]: type = LINEAR_SCALE_TYPE,
+    [`${attr}NoFallBack`]: noFallBack
+  } = props;
 
   let {[`${attr}Domain`]: domain} = props;
 
   // Return value-based scale if the value is assigned.
-  if (typeof value !== 'undefined') {
+  if (!noFallBack && typeof value !== 'undefined') {
```

Comment:
It's better not to write identifiers with a negation in them, otherwise you end up with double negatives in the code like `not noFallBack`. Would be better to reverse the logic and rename `noFallBack` to `hasFallback` or `isFallback`. Also `fallback` is a single word, so no need to upper case the `B`.